man/ukify.xml

```diff
@@ -211,6 +211,20 @@
         </para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--secureboot-certificate-dir=<replaceable>PATH</replaceable></option></term>
```

Comment:
would be good to explicitly mention that these two are required when using pesign, and ignored when using sbsign